src/components/lyne-clock/lyne-clock.tsx

```diff
@@ -0,0 +1,209 @@
+import {
+  Component,
+  Element,
+  h,
+  Prop
+} from '@stencil/core';
+
+import clockFaceSVG from './assets/sbb_clock_face.svg';
+import clockHandleHoursSVG from './assets/sbb_clock_hours.svg';
+import clockHandleMinutesSVG from './assets/sbb_clock_minutes.svg';
+import clockHandleSecondsSVG from './assets/sbb_clock_seconds.svg';
+
+let moveMinuteHand;
+
+@Component({
+  shadow: true,
+  styleUrl: 'lyne-clock.scss',
+  tag: 'lyne-clock'
+})
+
+export class LyneClock {
+
+  @Prop() public paused = false;
+  @Prop() public initialtime = 'now';
```

Comment:
Or i might have an even better proposal. Initialtime can have either a string (now) or a time-string (08:15:59). This could be type-checked with literals. But in my opinion this is a logical mismatch, since you're handling the two indescrete types differently. In my opinion, this would work more safely and would be better understandable:
1. if initialtime is not set, you handle it like it would be set to "now"
2. for the initialtime, you define a type:
  -> create a file `lyne-clock.d.ts` with the following: 
``` export type Time = `${number}:${number}:${number}`;```
  
  -> in your main file, import it like so: `import { Time } from './lyne-clock.d';`
  -> then use the type like so: `@Prop() public initialtime?: Time;`
  
That way, we are at least sure about getting a string which has the format "number:number:number". And we have a concise type for initialtime